rosbag2/src/rosbag2/converter.cpp

```diff
@@ -58,9 +60,8 @@ Converter::~Converter()
 std::shared_ptr<SerializedBagMessage> Converter::convert(
   std::shared_ptr<const rosbag2::SerializedBagMessage> message)
 {
-  auto ts = get_typesupport(topics_and_types_[message->topic_name], "rosidl_typesupport_cpp");
-  auto introspection_ts =
-    get_typesupport(topics_and_types_[message->topic_name], "rosidl_typesupport_introspection_cpp");
+  auto ts = topics_and_types_.at(message->topic_name).type_support;
```

Comment:
`at` might throw. Are we positive that the map is always filled at this point?

Reply:
Yes we are. `add_topic` must be (and is) called for every topic before a message of such a topic can be converted. Failing to do so would be a implementation error so a exception would be ok.